docs/release-process.md

```diff
@@ -4,13 +4,23 @@
 Regular releases are planned starting mid-February 2020. Until the API is
 stable we will be issuing v0.y versions.
 
-Today, the release process includes the following steps:
-* Creating release notes
-* Tagging the code
-* Creating the release using github
-* Announcing the release
-* Scheduling the next release
-* Documenting the release
+Today, the release process includes the following items:
+
+### Pre-release
+- [ ] Complete any API stability updates
```

Comment:
Do we need to add a section explaining the process here?

Reply:
I don't think so. In parallel to this I am working on a (partly) automated workflow to help identify and file issues related to api stability. I'm experimenting with it on my own fork of go-ceph at the moment. Once that is reviewed and merged I plan on returning to this document and updating it once more to reflect that new process. So you may want to consider this a placeholder for now.

Reply:
Ok.